pub/css/widget.css

```diff
@@ -52,16 +52,31 @@
     margin-left: 30px;
 }
 
-#w3tc-dashboard-widgets #normal-sortables .postbox {
-    width:360px;
+.widgets-container .meta-box-sortables {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	gap: 20px;
+	margin-bottom: 20px;
+	justify-content: center;
 }
 
-#w3tc-dashboard-widgets #side-sortables .postbox {
-    border:none;
-    box-shadow:none;
-    -webkit-box-shadow: none;
-    -webkit-border-radius: 0;
-    border-radius: 0;
+.widgets-container .meta-box-sortables .postbox {
+	width: calc(33.33% - 13.5px);
+	min-height: 300px;
+	box-sizing: border-box;
+	margin: 0;
+	//flex-grow: 1;
+}
+
+.widgets-container .meta-box-sortables #w3tc_imageservice,
+.widgets-container .meta-box-sortables #w3tc_bunnycdn {
+	//width: calc(50% - 10.5px);
```

Comment:
Replaced with comments. I had left this as I didn't want to lose the 1/2 column width calc portion in case changes were made or more widgets were added requiring a 50% width